### PR TITLE
Cypress ui setup

### PR DESCRIPTION
**Pull request by hitengpatel**
Hi @leighjohnson-friday and @dtaylor7,

This is my 1st Pull request on Cypress UI script. All the new files are located under test folder (cypress folder and cypress.json).
Apart from this there are few dependency updates on package.json. I hope this is not going to impact any code.

Thanks,
Hiten

### CODE REVIEW

File: test/cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// ***********************************************
```
Comment (dtaylor7) [inline test/cypress/support/commands.js:1]: this file can be removed?

**Review by dtaylor7 — CHANGES_REQUESTED** [review]
Excited to see this working!

There are a lot of comments throughout the code, i would be tempted to clean this up and only add comments where essential

File: package.json [hunk 2]
```diff
@@ -62,6 +64,7 @@
     "lodash": "^4.17.19",
     "mini-css-extract-plugin": "^0.9.0",
     "moment": "^2.27.0",
+    "mysql": "^2.18.1",
```
[the remark this answers is not in the document]
Comment (hitengpatel) [reply package.json:67]: Using Mysql2 now

File: test/cypress.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "supportFile": "cypress/support/index.js",
+    "nodeVersion": "system",
+    "videosFolder": "cypress/videos",
+    "pageLoadTimeout" :300000,
+    "defaultCommandTimeout": 30000,
+    "chromeWebSecurity": false,
+    "env":    
+    {"db":
+    {
+        "host": "localhost",
```
[the remark this answers is not in the document]
Comment (hitengpatel) [reply test/cypress.json:17]: Removed now and using current /config

**Review by dtaylor7 — APPROVED** [review]

**Review by leighjohnson-friday — APPROVED** [review]

**Pull request merged**